### PR TITLE
AAP-26615: Added info. about creating bundle secret to trust custom CA

### PR DESCRIPTION
**Pull request by sayjadha**
This PR tracks JIRA https://issues.redhat.com/browse/AAP-26615.

Changes made:
- Added procedure about creating bundle secret to trust custom CA.

### CODE REVIEW

File: lightspeed/modules/proc_create-connection-secrets.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 = Creating connection secrets
 
-Use this procedure to create secrets to connect to {PlatformName} and {ibmwatsonxcodeassistant}.
+You must create authorization secret to connect to {PlatformName}, and a model secret to connect to {ibmwatsonxcodeassistant}. If you need to trust a custom Certificate Authority, you must create a bundle secret.
```
Comment (hherbly) [inline lightspeed/modules/proc_create-connection-secrets.adoc:7]: Would change to "You must create *an* authorization secret"

File: lightspeed/modules/proc_create-connection-secrets.adoc [hunk 2]
```diff
@@ -86,4 +90,53 @@ image::aiconnect-auth-secret.png[{Example of an authorization secret]
...
+If you encounter a scenario where you need to trust a custom Certificate Authority, you can customize a few variables for the {LightspeedShortName} instance. Trusting a custom Certificate Authority enables the {LightspeedShortName} instance to access network services configured with SSL certificates issued locally, such as cloning a project from an internal Git server via HTTPS. 
+
+If you encounter the following error when syncing projects, it indicates that you need to customize the variables. 
+
+....
+fatal: unable to access 'https://private.repo./mine/ansible-rulebook.git': SSL certificate problem: unable to get local issuer certificate
+....
+
+.Procedure
+Use one of the following methods to create a custom bundle secret using CLI:
```
Comment (hherbly) [inline lightspeed/modules/proc_create-connection-secrets.adoc:108]: should be "*the* CLI"

**Review by hherbly — APPROVED** [review]
LGTM

File: lightspeed/modules/proc_create-connection-secrets.adoc (hunk 1 above)
Comment (sayjadha) [reply lightspeed/modules/proc_create-connection-secrets.adoc:7]: Updated

File: lightspeed/modules/proc_create-connection-secrets.adoc (hunk 2 above)
Comment (sayjadha) [reply lightspeed/modules/proc_create-connection-secrets.adoc:108]: Updated

**Pull request merged**